api/types/servers.go

```diff
@@ -18,11 +16,11 @@ type Server struct {
 }
 
 type ScriptChar struct {
-	ResourceType    string           `json:"resource_type" header:"RESOURCE_TYPE" show:"noshow,nolist"`
-	ID              string           `json:"id" header:"ID"`
-	Type            string           `json:"type" header:"TYPE"`
-	ParameterValues *json.RawMessage `json:"parameter_values" header:"PARAMETER_VALUES"`
-	TemplateID      string           `json:"template_id" header:"TEMPLATE_ID"`
-	ScriptID        string           `json:"script_id" header:"SCRIPT_ID"`
-	ExecutionOrder  int              `json:"execution_order" header:"EXECUTION_ORDER" show:"noshow,nolist"`
+	ResourceType    string                 `json:"resource_type" header:"RESOURCE_TYPE" show:"noshow,nolist"`
+	ID              string                 `json:"id" header:"ID"`
+	Type            string                 `json:"type" header:"TYPE"`
+	ParameterValues map[string]interface{} `json:"parameter_values" header:"PARAMETER_VALUES"`
```

Comment:
interface{} is comfortable for serializing, but problematic for deserializing. I understand that we just serialize, in our case...

Reply:
If I understand properly, yes.
Well, several fields come from json.RawMessage (an alias for [] byte). It is only used to print the results to the customer.